### ISSUE TITLE
Octant can't handle newly initialized metrics server

### ISSUE THREAD

**Issue by bryanl** [vmware-tanzu/octant#549]
If metrics server is installed while Octant is viewing workloads, it will error during the content generation proceeds until metrics are created for pods.

Error message looks like this:

```
2020-01-17T19:29:06.130-0500    ERROR   api/content_manager.go:103      generate content        {"client-id": "fde0caf4-3987-11ea-8d88-e450ebbc2e8c", "err": "generate content: collect workload cards: load workloads: get metrics for pod 'example-75778c488-655sb': get pod metrics: get pod metrics: podmetrics.metrics.k8s.io \"default/example-75778c488-655sb\" not found", "content-path": "workloads/namespace/default"}
github.com/vmware-tanzu/octant/internal/api.(*ContentManager).runUpdate.func1
        /Users/bryan/Development/projects/octant/internal/api/content_manager.go:103
github.com/vmware-tanzu/octant/internal/api.(*job).run.func1
        /Users/bryan/Development/projects/octant/internal/api/poller.go:111
```

**Comment by GuessWhoSamFoo** [comment]
To expand on this further, new pods added while the metrics server is installed will generate this error.

**Comment by bryanl** [comment]
Fixed in #573
